File: 782-JewelsAndStones/782-JewelsAndStones.py
# Last updated: 5/11/2026, 9:57:16 AM
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        jewels_set = set(jewels)

        return sum(stone in jewels_set for stone in stones)
        

File: 782-JewelsAndStones/782-JewelsAndStones.java
// Last updated: 5/11/2026, 9:57:09 AM
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        int n=0;
        for(int i=0; i<jewels.length(); i++){
            for(int j=0; j<stones.length(); j++){
                if(jewels.charAt(i) == stones.charAt(j)) n++;
            }
        }
        return n;
    }
}